app/screens/global_drafts/components/tabbed_contents/draftTabsHeader.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Tests for this file seem to be missing. I understand they will be coming in a different PR?

Reply:
Added the test